.changeset/silent-stilts-shush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query-builder': patch
+---
+
+Add option to create new query in query header actions
```

Comment:
this changeset description would be more relevant to legend-application-query. legend-query-builder doesn't really have the concept of creating new query

Reply:
True!